ecl/hqlcpp/hqlcpp.ipp

```diff
@@ -733,6 +733,7 @@ struct HqlCppOptions
     bool                implicitGroupHashDedup;
     bool                reportFieldUsage;
     bool                reportFileUsage;
+    bool                recordFieldUsage; // enables column-level security when turned on
```

Comment:
I would not mention column-level security in the comments throughout this PR. CLS is just one usage of recorded fieldUsage, there could be more. Maybe "enables recording field usage into the workunit"